src/js/constants/message.js

```diff
@@ -3,3 +3,9 @@ export const SEND_STATUS = {
     FAILED: 'failed',
     SENT: 'sent'
 };
+
+export const LOCATION_ERRORS = {
```

Comment:
I only see `PERMISSION_DENIED` used, are these codes generated by the browser or are we supposed to be using these values ourselves?